pom.xml

```diff
@@ -28,12 +28,12 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.4</version>
+        <version>3.13</version>
         <relativePath />
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-cps</artifactId>
-    <version>2.54-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```

Comment:
Because I haven't been paying enough attention - how does this work with `mvn release:prepare release:perform`?

Reply:
[Guide](https://github.com/jenkinsci/incrementals-tools/blob/master/README.md#running-maven-releases)